tests/docker/run_tests.sh

```diff
@@ -25,6 +25,6 @@ die() {
 }
 
 if ${SCRIPT_DIR}/ducker-ak ssh | grep -q '(none)'; then
-    ${SCRIPT_DIR}/ducker-ak up -n "${KAFKA_NUM_CONTAINERS}" || die "ducker-ak up failed"
+    ${SCRIPT_DIR}/ducker-ak up -j 'openjdk:11' -n "${KAFKA_NUM_CONTAINERS}" || die "ducker-ak up failed"
```

Comment:
These changes should be reverted prior merge.
Keeping it in PR just to show that all tests executed on JDK11.

Reply:
Could you remove this part since we are close to merge it?

Reply:
Reverted.